freshmaker/handlers/koji/rebuild_images_on_rpm_advisory_change.py

```diff
@@ -295,7 +296,8 @@ def _record_batches(self, batches, event, builds=None):
 
                 db.session.commit()
 
-                if state != ArtifactBuildState.FAILED.value:
+                is_unpublished_exception = is_in_unpublished_exceptions(image)
+                if state != ArtifactBuildState.FAILED.value and not is_unpublished_exception:
```

Comment:
To keep the same behavior, could you change the if statement below instead to something like this?
```python
if image["generate_pulp_repos"] or not (image["published"] or is_in_unpublished_exceptions(image)):
```